runtime/jcl/common/mgmtthread.c

```diff
@@ -1120,18 +1111,19 @@ handlerMonitorWaited(J9HookInterface** hook, UDATA eventNum, void* eventData, vo
  * @retval JNI_ENOMEM OutOfMemoryError is set
  * @retval JNI_ERR FindClass() or GetMethodID() failed, an exception is set
  */
-static jint
-initIDCache(JNIEnv *env)
+void JNICALL
+Java_openj9_management_internal_IDCacheInitializer_initIDCache (JNIEnv *env, jclass jcls)
 {
 	jclass oom;
```

Comment:
While you're here, please initialize all locals in their declaration.

Reply:
done.